java/src/main/native/src/row_conversion.cu

```diff
@@ -100,31 +94,26 @@ namespace detail {
  * complicating factors. They are spelled out below:
  *
  * Row Batches:
- * The row data has to fit inside a
- * cuDF column, which limits it to 2 gigs currently. The calling code attempts
- * to keep the data size under 2 gigs, but due to padding this isn't always
- * the case, so being able to break this up into multiple columns is necessary.
- * Internally, this is referred to as the row batch, which is a group of rows
- * that will fit into this 2 gig space requirement. There are typically 1 of
- * these batches, but there can be 2.
+ * The row data has to fit inside a cuDF column, which limits it to 2 gigs
+ * currently. The calling code attempts to keep the data size under 2 gigs,
+ * but due to padding this isn't always the case, so being able to break
+ * this up into multiple columns is necessary. Internally, this is
+ * referred to as the row batch, which is a group of rows that will fit
+ * into this 2 gig space requirement. There are typically 1 of these
+ * batches, but there can be 2.
```

Comment:
It seems that the comments in this file are line-broken manually. If using clang-format in cudf (format-on-save), you can typically write very long lines then the formatter will break line for you.